packages/mariadb/rollup.config.ts

```diff
@@ -0,0 +1,24 @@
+import { resolve } from 'path';
+import cleaner from 'rollup-plugin-cleaner';
+import typescript from 'rollup-plugin-typescript2';
+import versionInjector from 'rollup-plugin-version-injector';
+
+export default {
+  input: 'src/index.ts',
+  output: [
+    {
+      file: './dist/index.js',
+      format: 'cjs',
+      exports: 'named',
+      sourcemap: true
+    },
+    {
+      file: './dist/index.mjs',
+      format: 'es',
+      exports: 'named',
+      sourcemap: true
+    }
+  ],
+  external: [],
```

Comment:
This needs to list external dependencies, check mongoose for an example or check the warnings when you build this package